README.md

```diff
@@ -21,7 +21,7 @@ Hudi manages the storage of large analytical datasets on DFS (Cloud stores, HDFS
 
 <https://hudi.apache.org/>
 
-[![Build Status](https://travis-ci.org/apache/incubator-hudi.svg?branch=master)](https://travis-ci.org/apache/incubator-hudi)
+[![Build Status](https://travis-ci.org/apache/hudi.svg?branch=master)](https://travis-ci.org/apache/incubator-hudi)
```

Comment:
also link to. `https://travis-ci.org/apache/hudi` for the second url ?

Reply:
yes, done, it is outdated.